src/components/MentorList/MentorList.jsx

```diff
@@ -1,98 +1,85 @@
-import { useState, setState, useEffect, useRef, useLayoutEffect } from 'react'
-import useMentors from '../../hooks/use-mentors'
-import useMentorEvents from '../../hooks/use-mentor-events'
+import { useState, useEffect } from 'react'
+import MentorCard from '../MentorCard/MentorCard'
+import useSkills from "../../hooks/use-skills";
 import './MentorList.css'
 
-function MentorList({ activeEvent,
-    mentorsToAdd,
-    onMentorsAdd }) {
+// // --- Needs authentication handling to check token belongs to staff ---///
 
-    const { mentors, isMentorsLoading, isMentorsError, refreshComp } = useMentors()
-    const { mentorevents, isMentorEventsLoading, isMentorEventsError } = useMentorEvents()
+function MentorList({
+    activeEvent, 
+    allMentors,
+    mentorsToAdd, 
+    onMentorsAdd, 
+    allMentorEvents,
+    mentorsToRemove, 
+    onRemoveMentors }) {
 
     const [searchTermSkill, setSearchTermSkill] = useState("All")
     const [searchTermLocation, setSearchTermLocation] = useState("All")
     const [searchTermRank, setSearchTermRank] = useState("All")
 
     const [filteredMentors, updatFilteredMentors] = useState()
+    const [currentEventConfirmedMentors, setCurrentEventConfirmedMentors ] = useState()
 
-
-    // Some of these should be grabbed from API (skills)
-    const skills = ["Python", "Django", "DRF", "React", "Javascript", "Front-end", "Back-end", "HTML-CSS"]
+    
+    // TODO Some of these should be grabbed from API (skills)
```

Comment:
we can probably remove this comment now that we're grabbing skills from the database

Reply:
Thanks for switching these over! :D